components/openzwave-adapter/src/lib.rs

```diff
@@ -304,9 +312,12 @@ impl OpenzwaveAdapter {
                         include_map.push(include_setter_id.clone(), controller);
 
                         box_manager.add_channel(Channel {
-                            kind: ChannelKind::ZwaveInclude,
-                            supports_send: true,
-                            ..Channel::empty(&include_setter_id, &service_id, &adapter_id)
+                            implements: TaxoId::new("zwave/include"),
```

Comment:
ugh, `implements` is reserved keyword in javascript, not a blocker, just kind of inconvenient sometimes.